articles/connections/passwordless/sms-gateway.md

```diff
@@ -1,66 +1,74 @@
 ---
-title: Send One-time Codes via Your Own SMS Gateway
+title: Set Up SMS Gateway for Passwordless Connections
 topics:
     - connections
     - passwordless
     - sms
 contentType: how-to
 useCase: customize-connections
 ---
-# Send One-Time Codes via Your Own SMS Gateway
+# Set Up SMS Gateway for Passwordless Connections
 
-By default, the SMS connection uses Twilio to send the one-time code. If you already have your own infrastructure to send text messages, you cannot or might not want to use Twilio.
+If you are using a Passwordless connection, this guide will show you how you can use custom infrastructure that you've built (instead of Twilio) to send out one-time use codes via SMS messages to users who are logging in.
 
-Instead, you'll want your own infrastructure to handle the delivery of one-time codes.
+1. Get an [Access Token for Management API](/api/management/v2/tokens). You will need this to make calls to the Management API to update your Passwordless connection.
 
-## Configure your SMS Gateway
+2. Obtain information about the Passwordless SMS connection.
```

Comment:
Maybe give a little context in this step so they don't do their own thing and miss the note about copying all the options below? This doc isn't in the how-to format, but it does something similar (tells them up front why they're doing the step):
https://auth0.com/docs/connections/pass-parameters-to-idps#example-wordpress